lms/static/js/edxnotes/views/shim.js

```diff
@@ -58,6 +58,31 @@ define([
         return (timeToExpiry > 0) ? timeToExpiry : 0;
     };
 
+    Annotator.Plugin.Tags.prototype.pluginInit = _.compose(
+        function() {
+            // Add screen reader label for edit mode.
+            var index = this.annotator.editor.fields.length -1;
+            var tagsField = 'annotator-field-' + index;
+            $('<label class="sr" for='+ tagsField +'>' + _t('Edit tags') + '</label>').insertBefore(
+                $('#'+tagsField, this.wrapper)
+            );
+            return this;
+        },
+        Annotator.Plugin.Tags.prototype.pluginInit
+    );
+
+    Annotator.Plugin.Tags.prototype.updateViewer = _.compose(
```

Comment:
The problem is that the code is updating the prototype, so when ``Annotator.Plugin.Tags.prototype.updateViewer`` gets evaluated it will be the new version. I think that ``_.compose()`` is the right approach here so that we can update the prototype while still being able to call the old one.